public/css/mixin/progress-bar.less

```diff
@@ -9,23 +9,32 @@
 
     .above {
       padding: ~"calc(1em + 1px) 10%";
+
+      .now {
+        width: auto;
```

Comment:
That's not required I think. It doesn't get a width otherwise and `auto` is the default anyway. At least the `start` and `end` wrappers also don't have this.